SUI/SUI/ContentView.swift

```diff
@@ -5,14 +5,180 @@
 import SwiftUI
 
 struct ContentView: View {
+    //    MARK: - Private Constatns
+    private enum Constants {
+        static let onOffBlueToothPickerValue = ["Выкл.", "Вкл."]
```

Comment:
```suggestion
        static let onOffPickerValues = ["Выкл.", "Вкл."]
```
массивы с с на конце, и наверное я бы не указывала, что это конкретно блютуз, вдруг тебе где то еще Пикер с вкл выкл добавится